week5/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+import Button from "./Buttoninp";
+function App() {
+  const [newItem, setNewItem]= useState("");
+  const [items, setItems] = useState([]);
+ function additem(){
+  if (!newItem){
+    alert("please input item!");
+    return;
+  }
+  const item = {
+    id: Math.floor(Math.random() * 1000),
+    value: newItem
+  };
```

Comment:
the naming here is a bit off newItem is a string (not an object like item) so it is a bit confusing